packages/web-twig/src/Resources/components/Alert/README.md

```diff
@@ -29,13 +29,14 @@ Without lexer:
 
 ## API
 
-| Prop name     | Type                                         | Default   | Required | Description                |
-| ------------- | -------------------------------------------- | --------- | -------- | -------------------------- |
-| `class`       | `string`                                     | `null`    | no       | Custom CSS class           |
-| `color`       | [Emotion Color dictionary][dictionary-color] | `success` | no       | Color variant              |
-| `elementType` | `string`                                     | `div`     | no       | HTML tag to render         |
-| `iconName`    | `string`                                     | `null`    | no       | Icon used in Alert         |
-| `isCentered`  | `bool`                                       | `false`   | no       | If true, Alert is centered |
+| Prop name          | Type                                         | Default   | Required | Description                |
+| ------------------ | -------------------------------------------- | --------- | -------- | -------------------------- |
+| `color`            | [Emotion Color dictionary][dictionary-color] | `success` | no       | Color variant              |
+| `elementType`      | `string`                                     | `div`     | no       | HTML tag to render         |
+| `iconName`         | `string`                                     | `null`    | no       | Icon used in Alert         |
+| `isCentered`       | `bool`                                       | `false`   | no       | If true, Alert is centered |
+| `UNSAFE_className` | `string`                                     | -         | no       | Custom root class name     |
```

Comment:
Maybe it would be better to only add a note and a link here to another page with documentation about escape hatches. See https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-react/README.md#escape-hatches.

E.g. what about moving this part to root readme or somewhere else because it should be the same in both packages?